nautobot/core/graphql/schema.py

```diff
@@ -16,6 +16,7 @@
     generate_relationship_resolver,
     generate_restricted_queryset,
     generate_attrs_for_schema_type,
+    generate_computed_field_resolver,
```

Comment:
I know the others weren't alpha-sorted when you got here, but please alpha sort these!